README.md

```diff
@@ -81,6 +81,10 @@ This extension collects usage data in order to help us improve your experience.
 
 For troubleshooting SQL Client issues, You can enable verbose logging by setting the `AzureFunctions_SqlBindings_VerboseLogging` app setting (in your `*.settings.json` file) to '1', 'true' or 'yes';
 
+### Logs
+
+Logs for function apps deployed in Azure can be viewed in the Function's monitoring tab. If logs are not showing in the monitoring tab, they can also be found in the Function App's [Log Stream](https://learn.microsoft.com/azure/azure-functions/streaming-logs?tabs=azure-portal) or in [Application Insights](https://learn.microsoft.com/azure/azure-functions/analyze-telemetry-data#view-telemetry-in-application-insights). More information on logging can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitoring).
```

Comment:
let's be consistent with casing here "Function Apps" everywhere would be great.

Reply:
Updated to lower case to be consistent with MSDocs.